.github/workflows/test-application.yaml

```diff
@@ -43,12 +46,15 @@ jobs:
 
     services:
       mysql:
-        image: mysql:5.7
+        image: mysql:8.3
         env:
           MYSQL_ROOT_PASSWORD: root
           MYSQL_DATABASE: phpcr_tests
         options: >-
           --health-cmd "mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=5
+        steps:
+          - name: start
+            run: docker run -d mysql:8.3 --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci
```

Comment:
found this in https://github.com/orgs/community/discussions/25086

after trying 10 other things. @derrabus by any chance would you know how i can tell github actions what charset to use? it seems to default to utf8mb3 (man i hate stupid mysql collations mess)

Reply:
Does it still work for MySQL 5.7 though? I mean, you don't have to upgrade to 8.3 in order to support DBAL 4. You can still do that in a follow-up.

Reply:
@dbu i am not sure if it solves the probleme, but if i read the example you shared correct, the `steps` is maybe here correct?
https://github.com/jackalope/jackalope-doctrine-dbal/pull/440/files#diff-97081780ba6b676e25ca4729b13035dd4d3fd5f010dd76151028ae821707d3cdR69

But ci is not running on my fork, and runs need approvals

Reply:
also, dbal 4 seems to incorrectly report latin1 as encoding, when in reality it should be utf8. the PHP 8.0 builds install dbal 3.8.1, the PHP 8.1-3 install dbal 4.